packages/fxa-content-server/app/scripts/models/auth_brokers/oauth-redirect.js

```diff
@@ -345,7 +354,10 @@ export default BaseAuthenticationBroker.extend({
       this.clearOriginalTabMarker();
       return this.getOAuthResult(account).then(result => {
         result = _.extend(result, additionalResultData);
-        return this.sendOAuthResultToRelier(result);
```

Comment:
The way you reordered the metrics flush and sending the result to the relier makes much more sense. 👍